.github/workflows/build-and-push-to-ghcr.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and push to GHCR
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
This isn't needed if you're not building cross-arch.

Reply:
ok